lib/dolly/property.rb

```diff
@@ -4,7 +4,7 @@ class Property
     attr_accessor :name
     attr_reader :class_name, :default
 
-    CANT_CLONE = [NilClass, TrueClass, FalseClass, Fixnum].freeze
+    CANT_CLONE = [NilClass, TrueClass, FalseClass, Integer].freeze
```

Comment:
I would argue that you should still use fixnum

```ruby
> 1.frozen?
=> true
[2] pry(main)> 1.5.frozen?
=> true

Reply:
```ruby
> 1.5.clone
TypeError: can't clone Float

Reply:
well it actually looks like Float is not a fixnum, so that should be added too

Reply:
Fixnum is depracated in rails > 5.0 we need to remove it, and yeah we need to add float
I dont thing integer is needed

Reply:
does this change in rails 5? 

```ruby
> 1.clone
TypeError: can't clone Fixnum

Reply:
if we want a rails 5 branch. this should be a new major version. this wont work with rails 4, so it is a breaking change that requires a major version

Reply:
yeah it change > `1.clone` works on rails 5 with no error... I dont think this particular change affect us that much we can handle it and make it backwards compatible

Reply:
This change does affect us a lot. any default property that is an integer or a float would be affected

Reply:
this code is important for default properties to work

Reply:
And you don't think we can handle it in a way is compatible in both Tails 4 and 5?

Reply:
if there is a change to the inheritance models of the underlying rails/ruby code, I dont think so.  Feel free to prove me wrong. I feel like this is a change that will not be compatable with rails 4 if `Fixnum` does not exist